mage/changelog.md

```diff
@@ -4,6 +4,12 @@ title: Changelog
 sidebar_label: Changelog
```

Comment:
you will probably have to resolve conflicts with this file... if i were you, i would delete this file changes from this PR, and add what you want to the changelog in this PR

https://github.com/memgraph/docs/pull/644